tests/test_decorator.py

```diff
@@ -28,47 +28,34 @@ def foo(a: int, b: int):
 
     with pytest.raises(ValidationError) as exc_info:
         foo()
-
     assert exc_info.value.errors() == [
-        {'loc': ('a',), 'msg': 'field required', 'type': 'value_error.missing'},
-        {'loc': ('b',), 'msg': 'field required', 'type': 'value_error.missing'},
+        {'input': {}, 'loc': ('a',), 'msg': 'Field required', 'type': 'missing'},
+        {'input': {}, 'loc': ('b',), 'msg': 'Field required', 'type': 'missing'},
     ]
 
     with pytest.raises(ValidationError) as exc_info:
         foo(1, 'x')
-
     assert exc_info.value.errors() == [
-        {'loc': ('b',), 'msg': 'value is not a valid integer', 'type': 'type_error.integer'}
+        {
+            'input': 'x',
+            'loc': ('b',),
+            'msg': 'Input should be a valid integer, unable to parse string as an ' 'integer',
+            'type': 'int_parsing',
+        }
     ]
 
-    with pytest.raises(ValidationError) as exc_info:
+    with pytest.raises(TypeError, match='2 positional arguments expected but 3 given'):
```

Comment:
My understanding is that we no longer want to produce ValidationError's for these signature-related issues. If that's right, I think this should be good. I could also drop these tests that only have TypeError checks now if desired, or if we _do_ still want to produce ValidationError's for these, I can undo these changes and put them back to xfailing. 

(There are also a number of tests here that were passing but just needed updates to the error formats; we should at least merge those.)

Reply:
> My understanding is that we no longer want to produce ValidationError's for these signature-related issues

Are we just calling the function and letting it fail or something else?

Reply:
Yeah I think these type errors are just what you'd get from calling the function directly.